packages/ui/Resizable/useResize.js

```diff
@@ -0,0 +1,51 @@
+import {forward} from '@enact/core/handle';
+
+import {useCallback, useContext, useEffect, useRef} from 'react';
+
+import ResizeContext from './ResizeContext';
+
+/**
+ * A custom hook which returns handler that indicates a resize is necessary
+ * @function useResize
+ * @memberof ui/Resizable
+ * @ui
```

Comment:
Let's rewrite this like 

```suggestion
 *
 * @param ...
 * @returns ...
 * @private
```